cmd/scollector/main.go

```diff
@@ -34,6 +34,7 @@ var (
 	flagFilter          = flag.String("f", "", "Filters collectors matching this term, multiple terms separated by comma. Works with all other arguments.")
 	flagList            = flag.Bool("l", false, "List available collectors.")
 	flagPrint           = flag.Bool("p", false, "Print to screen instead of sending to a host")
+	flagHTTP            = flag.Bool("http", false, "Use HTTP instead of TCP")
```

Comment:
The default behaviour should not be changed. Reverse this logic and add a `tcp` flag that defaults to false.